components/org.wso2.carbon.identity.api.server.api.resource/org.wso2.carbon.identity.api.server.api.resource.v1/src/main/java/org/wso2/carbon/identity/api/server/api/resource/v1/impl/ScopesApiServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.api.server.api.resource.v1.impl;
+
+import org.wso2.carbon.identity.api.server.api.resource.v1.ScopesApiService;
+import org.wso2.carbon.identity.api.server.api.resource.v1.core.ServerAPIResourceManagementService;
+
+import javax.ws.rs.core.Response;
+
+/**
+ * Implementation of scopes API.
+ */
+public class ScopesApiServiceImpl implements ScopesApiService {
+
+    ServerAPIResourceManagementService serverAPIResourceManagementService =
+            ServerAPIResourceManagementService.getInstance();
```

Comment:
Shall we go with    @Autowired annotation and let it be resolve and inject the corresponding bean
   @Autowired
    private ServerAPIResourceManagementService serverAPIResourceManagementService;